### ISSUE TITLE
Questions about prediction

### ISSUE THREAD

**Issue by CoinCheung** [Megatvini/DeepFaceForgeryDetection#5]
Hi, 

Thanks for releasing this, after going through the code, I found that, in the following line, you seem to predict the frame to be fake as long as the score is greater than 0, why would you do that and is the prediction real only when the score is 0 ?

https://github.com/Megatvini/DeepFaceForgeryDetection/blob/109e03abc3f3886e56539d7b5220807126295856/src/evaluate.py#L80

**Comment by Megatvini** [comment]
Hey,
I think what you are missing is that the model don't include the last activation layer, so it outputs **logits** instead of **probabilities**. 
That is also why I am using nn.BCEWithLogitsLoss() instead of regular nn.BCELoss()

**Comment by tranvansanghust** [comment]
@CoinCheung Did you successfully run prediction???
